docs/en/index.md

```diff
@@ -203,7 +203,7 @@ You can find the [source code to snap and snapd on github][source-code-to-snap-a
 
 ### Tutorials
 
-There is also the [Ubuntu Tutorials][ubuntu-tutorials] site that covers tutorials about creating and working with snaps, creating new Ubuntu images (both classic Ubuntu and Core), advanced snap usage such as snapping up applications using various programming languages, integrating a continuous integration (CI) system into working with snaps, and much more.
+There is also the [Ubuntu Tutorials][ubuntu-tutorials] site that covers tutorials about creating and working with snaps, creating new Ubuntu images (both Ubuntu Classic and Core), advanced snap usage such as snapping up applications using various programming languages, integrating a continuous integration (CI) system into working with snaps, and much more.
```

Comment:
Probably better with links, but I think the list of tutorials is difficult to parse: `There is also the [Ubuntu Tutorials][ubuntu-tutorials] site. Here you'll find tutorials about creating and working with snaps, building new Ubuntu images (both Ubuntu classic and Core), advanced snap usage, using continuous integration (CI) and much more.`